ui/tests/index.html

```diff
@@ -38,10 +38,10 @@
       </div>
     </div>
 
-    <script src="/testem.js" integrity="" data-embroider-ignore></script>
     <script src="{{ rootURL }}assets/vendor.js"></script>
     <script src="{{ rootURL }}assets/test-support.js"></script>
     <script src="{{ rootURL }}assets/nomad-ui.js"></script>
+    <script src="/testem.js" integrity="" data-embroider-ignore></script>
```

Comment:
Now that we specify a framework in testem directly, it appears that script loading order matters.